### DISCUSSION TITLE
Discord Linj

### DISCUSSION THREAD

**Discussion by 7ank0v1c** [claunia/DiscImageChef#798] [General]
Hey guys, I was wondering where I could find the link to the AARU Data Preservation Suite discord server?

**Comment by robin-francois** [comment]
The Aaru Discord is not public. It is a private server for the technical committee, developers and stakeholders of Aaru. Digital archivists or users of Aaru in the context of a memory/GLAM institution can sometimes be invited to join.

For a public channel to discuss Aaru on Discord, please join the VGPC Discord and discuss on the aaru channel (under the "projects" category)

**Reply by 7ank0v1c** [reply]
Just seeing this now. Thanks 😊
